ui_framework/components/modal/__snapshots__/confirm_modal.test.js.snap

```diff
@@ -43,8 +44,9 @@ exports[`renders KuiConfirmModal 1`] = `
       </span>
     </button>
     <button
-      class="kuiButton kuiButton--primary"
+      class="kuiButton"
       data-test-subj="confirmModalConfirmButton"
+      type="primary"
```

Comment:
Looks like we'll need to update `confirm_modal.js`, which creates instances of `KuiButton`.